form.html

```diff
@@ -17,13 +17,13 @@ <h1>Quiz App</h1>
     </header>
     <main>
       <h2 class="headlineForm" title="cardCreation">Create a card</h2>
-      <form id="cardCreation" class="form">
-        <label for="your-question">Your Question</label>
-        <textarea id="your-question"></textarea>
-        <label for="your-answer">Your Answer</label>
-        <textarea id="your-answer"></textarea>
-        <label for="tag">Tag</label>
-        <input type="text" id="tag"></input>
+      <form id="cardCreation" class="form" data-js="form">
+        <label for="your-question">Your Question:</label>
+        <textarea required name="inputQuestion" class="form__textarea" id="your-question"></textarea>
+        <label for="your-answer">Your Answer:</label>
+        <textarea required name="inputAnswer" id="your-answer"></textarea>
```

Comment:
if you give a name its implied that it is an input element. You can keep it shorter with `name="answer"`